geps/gep-1897.md

```diff
@@ -0,0 +1,113 @@
+# GEP-1897: TLS from Gateway to Backend for ingress
+
+* Issue: [#1897](https://github.com/kubernetes-sigs/gateway-api/issues/1897)
+* Status: Provisional
+
+## TLDR
+
+This document specifically addresses the topic of conveying HTTPS from the Gateway
+dataplane to the backend (backend TLS termination), and intends to satisfy the single
+use case “As a client implementation of Gateway API, I need to know how to connect to
+a backend pod that has its own certificate”. TLS configuration can be a nebulous topic,
+so in order to drive resolution this GEP focuses only on this single piece of functionality.
+
+## Goals
+
+1. The solution should satisfy the following use case: the backend pod has its own
+certificate and the gateway implementation client needs to know how to connect to the
+backend pod. (Use case #4 in [Gateway API TLS Use Cases](#references))
+2. In terms of the Gateway API personas, the application developer should control the
+gateway to backend TLS settings, not the cluster operator, as requiring a cluster operator
+to manage certificate renewals and revocations would be extremely cumbersome.
```

Comment:
On client certificate - I don't think an API is appropriate. App code can ask for them ( using TLS protocol ), the API can't influence this. And the Gateway - if it can - should present its identity. App owner can't choose a random client cert.

Reply:
Also - Gateway SHOULD authenticate with either a JWT with audience or client cert or some other mean - so gateway added headers can be trusted, etc.

Reply:
I am referring to the application developer as a Gateway API persona, and thus the person who creates the HTTPRoute.  The second comment, iiuc is implementation dependent.